website_sale_credit_point/i18n/fr.po

```diff
@@ -0,0 +1,22 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* website_sale_credit_point
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 11.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-04-12 08:06+0000\n"
+"PO-Revision-Date: 2018-04-12 08:06+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: website_sale_credit_point
+#: model:ir.ui.view,arch_db:website_sale_credit_point.navbar_points
+msgid "<span>Credit balance: </span>"
+msgstr "<span>Solde disponible: </span>"
```

Comment:
Minor but in French colons always have a (non breaking) space in front of them (eg. "Solde disponible :")